cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/VolumeParameterConfig.java

```diff
@@ -60,4 +62,12 @@ public String toString() {
                 + ", maximumNumber=" + maximumNumber
                 + '}';
     }
+
+    private Integer calculateSupportedMaximumNumber(Integer maximumNumber) {
```

Comment:
Can it be `null`? If yes, the `>` comparison will result in an NPE. If not, `int` would be better.